test/integration_tests/long/test_load_executor.py

```diff
@@ -231,24 +230,32 @@ def test_should_rollback_if_video_load_fails(self):
                         do_not_print_exceptions=True,
                     )
 
+            # Load one correct file and one empty file
+            # one file should get added
+            with tempfile.TemporaryDirectory() as tmp_dir:
+                shutil.copy2(str(valid_videos[0]), tmp_dir)
+                shutil.copy2(str(empty_file.name), tmp_dir)
```

Comment:
Could I get a clarification why we skip the empty file?

Reply:
Currently, we are aborting the entire process when the load executor encounters a corrupted file. Post my changes, all the valid files will get loaded, and we are gonna log the corrupted files and ignore them